build-cwl-docker.sh

```diff
@@ -2,3 +2,10 @@
 set -e
 docker build --file=cwltool.Dockerfile --tag=commonworkflowlanguage/cwltool-module --target module .
 docker build --file=cwltool.Dockerfile --tag=commonworkflowlanguage/cwltool .
+
+version=$(git describe --tags)
+echo $version | grep -vq '\-' >& /dev/null
+if [ $? -eq 0 ];then
+    docker tag commonworkflowlanguage/cwltool_module commonworkflowlanguage/cwltool_module:$version
```

Comment:
Here we have `cwltool_module` but above it is `cwltool-module` (underscore vs. dash); this will likely fail